source/install/trouble_mysql.rst

```diff
@@ -0,0 +1,268 @@
+.. _mysql-installation-troubleshooting:
+
+MySQL Installation Troubleshooting
+----------------------------------
+
+Before you can run the Mattermost server, you must first install and
+configure a database. You can then start Mattermost by navigating to
+``/opt/mattermost/bin`` and entering the command ``sudo -u mattermost ./platform``.
+If the Mattermost server cannot connect to the database, Mattermost
+will fail to start. This section deals with MySQL database issues that
+you may encounter when you start up Mattermost for the first time.
+
+How you install MySQL varies depending upon which Linux distribution you
+use, but once MySQL is installed the configuration instructions are the
+same for all supported distributions. You must create a ``mattermost`` database
+and a ``mattermost`` database user. Failure to create these database
+objects or improperly referencing them from the Mattermost configuration
+file, ``/opt/mattermost/config/config.json``, prevents Mattermost from
+starting. The troubleshooting tips given here deal with these specific
+issues.
+
+Before proceeding, confirm that your MySQL server is running. You can do
+this by issuing the following command: ::
+
+    mysqladmin -u root -p status
```

Comment:
Is it possible to do this with the regular mysql command? I think this command needs to be installed separately from the regular MySQL client.

This also assumes that the MySQL server is running on the same machine which it likely won't be. We should include whatever options are required to check if a database on another machine is running.